keanu-python/build.gradle

```diff
@@ -43,7 +43,6 @@ task installDependencies {
 
 task preparePythonEnvironment {
     dependsOn(installPipenv)
-    dependsOn(pythonVersionInfo)
```

Comment:
fine by me. We only really needed this diagnostic information when we were first setting the python build up to run on the CI servers.